operators/Makefile

```diff
@@ -280,39 +280,60 @@ TESTS_MATCH ?= ""
 E2E_IMG ?= $(IMG)-e2e-tests:$(TAG)
 
 # Run e2e tests as a k8s batch job
-e2e: e2e-setup e2e-deploy-operators e2e-docker-image e2e-run
+e2e: docker-build docker-push e2e-docker-build e2e-docker-push e2e-run
 
-e2e-setup:
-	@ echo -- Setup e2e rbac
-	kubectl apply -f config/e2e/rbac.yaml
-
-e2e-deploy-operators:
-	@ echo -- Deploy operators to manage e2e namespace
-	@ $(MAKE) --no-print-directory MANAGED_NAMESPACE=e2e deploy
-
-e2e-docker-image:
-	@ echo -- Build and push e2e docker image
+e2e-docker-build:
 	docker build -t $(E2E_IMG) -f test/e2e/Dockerfile .
+
+e2e-docker-push:
 	docker push $(E2E_IMG)
 
 e2e-run:
-	@ echo -- Run e2e as a job
-	./hack/run-e2e.sh "$(E2E_IMG)" "$(TESTS_MATCH)"
+	@go run test/e2e/cmd/main.go run \
+		--operator-image=$(OPERATOR_IMAGE) \
+		--e2e-image=$(E2E_IMG) \
+		--test-regex=$(TESTS_MATCH)
+
+# targets to run e2e tests in a Kind cluster
+KIND_CLUSTER ?= "eck-e2e"
+
+e2e-kind: e2e-kind-create-cluster e2e-kind-load e2e-kind-run
+
+e2e-kind-create-cluster:
+	kind create cluster --name=$(KIND_CLUSTER) --config=config/e2e/kind.yaml
```

Comment:
I'm not very well familiar with `kind`, but it looks like there is no way to specify which k8s version should be used for e2e tests right now. I think we should be able to change version. Via parameter it can be done like `--image kindest/node:v1.14.1`, not sure about yml config

Reply:
Currently the E2E tests fail on Kind because they attempt to write to a host directory that is read-only. I did not try to investigate it further because I was only using Kind to quickly spin-up a Kubernetes cluster to test that my code works. 

For the future, Kind certainly seems like a promising way to quickly run tests on different K8S versions.

Reply:
So you are saying Kind does not work right now for your test, should we remove these merge targets then?

Reply:
Sure, I will remove them.